mediator/esmFldsExchange_coastal_mod.F90

```diff
@@ -236,6 +258,83 @@ subroutine esmFldsExchange_coastal_advt(gcomp, phase, rc)
       deallocate(S_flds)
     end if
 
+    ! ---------------------------------------------------------------------
+    ! to ice: atm fields
+    ! ---------------------------------------------------------------------
+    if (coastal_attr%atm_present .and. coastal_attr%ice_present) then
+      allocate(S_flds(14))
+      S_flds = (/'Sa_u10m',       & ! inst_zonal_wind_height10m
```

Comment:
@SmithJos13 You might want to change Sa_u10 and v10 as Sa_u and v since we are trying to keep the changes in CICE side minimal. Please also fallow the convention about indentations. BTW, all strings in the array needs to be same length. Otherwise, we could have issue with some compilers.